### PR TITLE
Remove ATTRIB URL and note the deprecation of ATTRIB usage

### PR DESCRIPTION
**Pull request by swcurran**
Per discussion of WG call -- 20220208, add note about ATTRIB deprecation, and that it will not have a URL format.

### CODE REVIEW

File: spec/10_did_operations.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ The following are the steps for assembling a DIDDoc from its inputs.
 2. The Indy network instance `namespace`, the [[ref: NYM]] `dest` and the [[ref: NYM]] `verkey` items are merged into a text template to produce a base DIDDoc.
     1. See the template in the [Base DIDDoc Template](#base-diddoc-template) section of this document.
     2. If there is no `diddocContent` item in the [[ref: NYM]], assembly is complete; return the DIDDoc and a success status.
+       1. For backwards compatibility to the `did:sov` DID Method, if there is no `diddocContent` iten, a client DID resolver **MAY** attempt to read the `endpoint` ATTRIB object related to the DID and add it to the DIDDoc. See the [Endpoint ATTRIB section (below)](#the-%22endpoint%22-attrib) for more details.
```
Comment (dbluhm) [inline spec/10_did_operations.md:46]: Typo
```suggestion
       1. For backwards compatibility to the `did:sov` DID Method, if there is no `diddocContent` item, a client DID resolver **MAY** attempt to read the `endpoint` ATTRIB object related to the DID and add it to the DIDDoc. See the [Endpoint ATTRIB section (below)](#the-%22endpoint%22-attrib) for more details.
```

File: spec/10_did_operations.md [hunk 2]
```diff
@@ -164,7 +165,7 @@ By default there is no key agreement section in an assembled DIDDoc. If the DID
 
 Prior to the definition of this DID Method, a convention on Indy ledgers to associate an endpoint to a [[ref: NYM]] involved adding an [[ref: ATTRIB]] ledger object with a `raw` value of contain the JSON for a name-value pair of `endpoint` and a URL endpoint, often an IP address.
 
-We recommend that anyone that is using that format prior to the availability of the `did:indy` DID Method update their [[ref: NYM]] on the ledger to use the `diddocContent` item as soon as possible.
+We strongly encourage anyone using the "ATTRIB endpoint" convention to update their [[ref: NYM]] on the ledger to use the `diddocContent` item as soon as possible, as the [[ref: ATTRIB]] is deprecated as of the availability of the `did:indy` DID Method.
```
Comment (dbluhm) [inline spec/10_did_operations.md:168]: Minor suggestion to improve readability; uses the same wording as is used in the attrib section
```suggestion
We strongly encourage anyone using the "ATTRIB endpoint" convention to update their [[ref: NYM]] on the ledger to use the `diddocContent` item as soon as possible, as the [[ref: ATTRIB]] is deprecated with the introduction of the `did:indy` DID Method.
```

**Review by dbluhm — COMMENTED** [review]
A couple minor suggestions

**Comment by swcurran** [comment]
DCO issues -- closing in favour of #48

**Pull request closed without merging**